waspc/src/Generator/WebAppGenerator/OperationsGenerator/ResourcesG.hs

```diff
@@ -0,0 +1,20 @@
+module Generator.WebAppGenerator.OperationsGenerator.ResourcesG
+    ( genResources
+    ) where
+
+import           Data.Aeson                                  (object)
+import qualified Path                                        as P
+
+import           StrongPath ((</>))
+import qualified StrongPath as SP
+import           Generator.FileDraft                         (FileDraft)
+import qualified Generator.WebAppGenerator.Common            as C
+import           Wasp                                        (Wasp)
+
+
+genResources :: Wasp -> [FileDraft]
+genResources _ = [C.makeTemplateFD tmplFile dstFile (Just tmplData)]
+  where
+    tmplFile = C.asTmplFile [P.relfile|src/operations/resources.js|]
+    dstFile = C.asWebAppFile $ [P.relfile|src/operations/resources.js|] -- TODO: Un-hardcode this by combining path to operations dir with path to resources file in it.
```

Comment:
I added this TODO to solve it before I finalize the PR, but now I think we can just leave it for later, there are more pressing things to tend to. Even if I un-hardcode this part, there are many more hardcoded parts.

Reply:
aha ok, didn't even notice it before :D Looks good to me :D